tests/PhpSpreadsheetTests/Worksheet/Table/Issue3569Test.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
issue number is 3659, not 3569 :)

Reply:
I agree, please go ahead and change the name of the test (and the class name inside the test).

Reply:
Actually, I have more changes. I will take care of the rename.